packaging/debian/control

```diff
@@ -0,0 +1,24 @@
+Source: cockpit-navigator
+Section: admin
+Priority: optional
+Maintainer: Utopia Maintenance Team <pkg-utopia-maintainers@lists.alioth.debian.org>
+Uploaders: Martin Pitt <mpitt@debian.org>,
+Build-Depends: debhelper-compat (= 13),
+               gettext (>= 0.19.7),
+               gettext (>= 0.21) | appstream,
```

Comment:
compared to podman, `gettext` is not included, nor `appsteam`

Reply:
But c-navigator already has a metadata file. However, all supported Debian and Ubuntu releases now have gettext ≥ 0.21, so these two lines should change to `gettext (>= 0.21)` (we can do the same on our other projects).